pages/contact.js

```diff
@@ -35,7 +35,7 @@ const contact = () => {
 						<div className='dropdown dropdown-hover  relative w-full p-3 rounded-box border border-[#B7B7B7] shadow-lg'>
```

Comment:
change from dropdown to collapse with background
[https://daisyui.com/components/collapse/](url)